tests/unit/test_clidriver.py

```diff
@@ -261,6 +261,15 @@ def test_error_logger(self):
         expected = {'log_level': logging.ERROR, 'logger_name': 'awscli'}
         self.assertEqual(driver.session.stream_logger_args[1], expected)
 
+    def test_ctrl_c_is_handled(self):
+        driver = CLIDriver(session=self.session)
+        fake_client = mock.Mock()
+        fake_client.list_objects.side_effect = KeyboardInterrupt
+        fake_client.can_paginate.return_value = False
+        driver.session.create_client = mock.Mock(return_value=fake_client)
+        rc = driver.main('s3 list-objects --bucket foo'.split())
+        self.assertEqual(rc, 130)
+
```

Comment:
No, there is no stderr to capture.

When a traceback gets all the way up the stack *python itself* will print the traceback for you before exiting.  It is impossible to test for stderr in this scenario in a unit test because we're not the one printing the traceback in the first place.

Take a look at my [latest commit](https://github.com/jamesls/aws-cli/commit/c09e9e7cd6f09c7bf5f993f9fcf76d13390f1f1f), where I add an integ test.  This is the only way to test this functionality if you want to verify that there's no Traceback in stdout/stderr.